search-sentiments-map/test/trends-test.js

```diff
@@ -0,0 +1,243 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+const chai = require('chai');
+const assert = chai.assert;
+const sinon = require('sinon');
+const trends = require('./../routes/trends').trends;
+const googleTrends = require('google-trends-api');
+const {Datastore} = require('@google-cloud/datastore');
+
+describe('Trends', function() {
+  describe('RetrieveGlobalTrendsForTimeRange', function() {
+    const CURRENT_DATA_TIME_RANGE_12_HOURS_MS = 12 * 60 * 60000;
+    const TIME_RANGE_1_HALF_MS = CURRENT_DATA_TIME_RANGE_12_HOURS_MS * 1.5;
+    const TIME_RANGE_3_MS = CURRENT_DATA_TIME_RANGE_12_HOURS_MS * 3;
```

Comment:
For these constants what Haylee told me before was to make one constant for the time range value i.e. THREE_TIME_RANGE = 3, and then just multiply THREE_TIME_RANGE * CURRENT_DATA_TIME_RANGE_12_HOURS_MS in the subtraction below, and I think @hayleejane3 can comment further, or correct me if I'm wrong.